src/frontend/screens/Onboarding/walletSelection/screens/scan/index.tsx

```diff
@@ -40,7 +40,9 @@ const ScanScreen = (props: ScanScreenProps) => {
         )}{' '}
         <a
           onClick={() =>
-            window.api.openExternalUrl('https://metamask.io/download/')
+            window.api.openExternalUrl(
+              'https://metamask.io/download/?utm_source=hyperplay'
```

Comment:
Would be good to use this in a exported variable like above to easy future changes if needed.

Reply:
sgtm. made the change 👍